website/pages/docs/connect/ca/index.mdx

```diff
@@ -19,9 +19,9 @@ storing and signing certificates. Consul ships with a
 root certificate and private key on the Consul servers. Consul also has
 built-in support for
 [Vault as a CA](/docs/connect/ca/vault). With Vault, the root certificate
-and private key material remain with the Vault cluster. A future version of
-Consul will support pluggable CA systems using external binaries.
+and private key material remain with the Vault cluster.
 
+Consul will support pluggable CA systems using external binaries.
```

Comment:
Do you want to completely remove this part too?

Reply:
Thanks for catching that. I must've accidentally pasted that before committing. I removed it as intended, and updated the PR.